README.md

```diff
@@ -86,6 +86,35 @@ Uses the same options as [`createContentType`](#createcontenttypeid--string-opts
 
 Deletes the content type with the provided id and returns `undefined`. Note that the content type must not have any entries.
 
+#### `transformEntries(config)`
+
+For the given content type, transforms all entries according to the user-provided `transformEntryForLocale` function. This function will be called for every entry and every locale with the `from` fields and the current locale. It is expected to return an object with the desired target fields and the value OR undefined, in which case this entry locale will be skipped. If `transformEntryForLocale` does return a value, it will be written to the current locale.
+Please also refer to [this example](./examples/12-transform-content.js).
+
+**`config : Object`** – Content transformation definition, with the following properties:
+- **`contentType : string`** _(required)_ – Content type ID
+- **`from : array`** _(required)_ – Array of the source field IDs
+- **`to : array`** _(required)_ – Array of the target field IDs
+- **`transformEntryForLocale : function (fields, locale): object`** _(required)_ – Transformation function to be applied.
+- **`shouldPublish : boolean`** _(optional)_ – Publish entries after transform (default _true_)
+
+#### `deriveLinkedEntries(config)`
+
+For the given content type, derives new entries from existing ones and sets up the new ones as a reference on the existing one according to the user-provided `deriveEntryForLocale` function. This function will be called for every entry and every locale with the `from` fields and the current locale. It is expected to return an object with the desired target fields and the value OR undefined, in which case this entry locale will be skipped. 
```

Comment:
Another suggestion:

`For each entry of the given content type, derives a new entry and sets up a reference to it. The content of the new entry is generated by the user provided 'deriveEntryForLocale` function`

Some of the comments I made for `transformEntryForLocale` also apply to this function:

- `undefined`
- `skipped` vs `untouched`
- Clarification of `fields` and `locale` in `deriveEntryForLocale`